applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -446,11 +446,31 @@ int le_audio_volume_mute(void)
 
 int le_audio_play(void)
 {
+	int ret;
+
+	ret = bt_audio_capability_set_available_contexts(
+		BT_AUDIO_DIR_SINK, BT_AUDIO_CONTEXT_TYPE_MEDIA | BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED);
```

Comment:
Should we set BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED? Doesnt this mean "we can receive everything". Maybe I remember wrong.

Reply:
We used to support both MEDIA and UNSPECIFIED, not sure if we should use BT_AUDIO_CONTEXT_TYPE_PROHIBITED or not when pausing and then just MEDIA when playing. I will find out and update accordingly

Reply:
We can't set the PROHIBITED at least, so I think we have to have UNSPECIFIED because we can't set it to nothing

Reply:
BT_AUDIO_CONTEXT_TYPE_UNSPECIFIED is a madatory configuration.